argilla/docs/how_to_guides/export.md

```diff
@@ -0,0 +1,156 @@
+---
```

Comment:
in order for these files to show up in toc and navigation you should also update the `mkdocs.yaml`